### ISSUE TITLE
Best way to detect node had loaded?

### ISSUE THREAD

**Issue by simonwjackson** [wavesoft/dot-dom#18]
Any way to react once the vnode has been rendered to the screen?

This doesn't seem to work:

```javascript
function utterance(props, state, setState) {
  let inner = 'hey'
  let obj = {
    onload () {
      this.setAttribute("data-id", props.id)
    }
  }
  return H('div', obj, inner);
}

R(H(utterance), document.body)
```

**Comment by wavesoft** [comment]
You might have to wait for the lifecycle methods. I am currently thinking for the best approach to it,  so be aware that the API might change. Nonetheless, you can try to use the build from here https://github.com/wavesoft/dot-dom/blob/ba038fbda06ad17d3deb983605fd406f3060851b/dotdom.min.js

An you can see an example of the new API in the PR here https://github.com/wavesoft/dot-dom/pull/12

**Comment by simonwjackson** [comment]
Wow, that's fantastic! Thanks again!

**Comment by wavesoft** [comment]
This will be resolved with #12 please comment there if there are any other issues 👍
